phoenix-core/src/it/java/org/apache/phoenix/schema/stats/StatsCollectorIT.java

```diff
@@ -38,6 +38,7 @@
 import java.util.Properties;
 import java.util.Random;
 
+import com.google.common.collect.Lists;
```

Comment:
You should also add a test where you row a few rows and then  delete them and call update statistics with and without include deleted cells and verify that they guide point width takes into account delete cells correctly.